### ISSUE TITLE
Enhancement: document process to use alternatives to UDP for messaging

### ISSUE THREAD

**Issue by BobBooth** [Esri/military-apps-library-java#57]
Point to what code needs to be updated to use custom network protocols (e.g.on low bandwidth radio networks) and give example.

**Comment by jrweakland** [comment]
@garys-esri - Has this been addressed in the past?

**Comment by garys-esri** [comment]
No, not yet. An advanced developer would be able to do this work without too much trouble, but we should document it sometime to make it crystal clear.
